### PR TITLE
changed oadp validation bullets

### PR DESCRIPTION
**Pull request by jc-berger**
See Jira: https://issues.redhat.com/browse/ACM-12008

### CODE REVIEW

File: business_continuity/backup_restore/backup_validate.adoc [hunk 1]
```diff
@@ -11,11 +11,14 @@ The `backup-restore-enabled` policy includes a set of templates that check for t
...
+* When you enable the backup component on the `MultiClusterHub`, the cluster backup and restore operator Helm chart can automatically install the OADP operator in the `open-cluster-management-backup` namespace. You can also manually install the OADP operator in the namespace. 
+* The `OADP-channel` that you selected for manual installation must match or exceed the version set by the {product-title-short} backup and restore operator Helm chart. 
+* Since the OADP Operator and Velero Custom Resource Definitions (CRDs) are `cluster-scoped`, you cannot have multiple versions on the same cluster. Therefore, install the same version in the `open-cluster-management-backup` namespace and any other namespaces. 
+* Use the following templates to check for availability and validate the OADP installation: 
+** `oadp-operator-exists`: Use this template to verify if the OADP operator 
+is installed in the `open-cluster-management-backup` namespace.
+** `oadp-channel-validation`: Use this temmplate to ensure the OADP operator versino in the `open-cluster-management-backu`p` namespace matches or exceeds the version set by the {product-title-short} backup and restore operator.
```
Comment (sahare) [inline business_continuity/backup_restore/backup_validate.adoc]: Please correct the typos: temmplate, versino and `open-cluster-management-backu`p`

**Review by sahare — CHANGES_REQUESTED** [review]

**Review by sahare — APPROVED** [review]
lgtm, thanks.

File: business_continuity/backup_restore/backup_validate.adoc [hunk 2]
```diff
@@ -11,11 +11,14 @@ The `backup-restore-enabled` policy includes a set of templates that check for t
...
-* In the following examples, the backup and restore operator could run with wrong CRDs resulting in erroneous behavior: 
-** {product-title-short} has many installed versions of OADP. 
-** If the OADP version installed by `MultiClusterHub` is uninstalled, and you manually install a different version.
+* When you enable the backup component on the `MultiClusterHub`, the cluster backup and restore operator Helm chart can automatically install the OADP operator in the `open-cluster-management-backup` namespace. You can also manually install the OADP operator in the namespace. 
+* The `OADP-channel` that you selected for manual installation must match or exceed the version set by the {product-title-short} backup and restore operator Helm chart. 
+* Since the OADP Operator and Velero Custom Resource Definitions (CRDs) are `cluster-scoped`, you cannot have multiple versions on the same cluster. Therefore, install the same version in the `open-cluster-management-backup` namespace and any other namespaces.
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_validate.adoc]: I would remove Therefore (it's extra and not needed) and just be very imperative here:

_You must install the same version in the `open-cluster-management-backup` namespace and any other namespaces._

**Review by swopebe — APPROVED** [review]
Looks great; one recommendation.

**Pull request merged**